Usage Console App (#4180)
SoftwareAG/c8y-docs: content/console/environments.md

---
title: Environments
layout: bundle
sector:
  - platform_administration
weight: 30
---

> The Environments page gives you a high-level summary of each of the environments associated with your account.

Reviewer: ```suggestion
The **Environments** page provides a high-level summary of each of the environments associated with your account.
```

<img src="/images/console/environments.png" alt="Environments">

In the above example, the customer has four environments under their account. Each environment can be viewed by clicking the respective tab at the top of the page, where a variety of details are shown, including the domain, tenant ID, and provisioning date. 

> Under the **Subscribed Add-Ons** section, you can view which Add-Ons have been subscribed to that specific environment. For the Preview, Add-Ons are subscribed manually by {{< company-c8y >}}. If you want to add a new Add-On to an environment, you should reach out to your Customer Success Manager or Account Executive, else you should contact [product support](/additional-resources/contacting-support/). A short description of each Add-On is available.

Reviewer: ```suggestion
Under the **Subscribed add-ons** section, you can view which add-ons have been subscribed to that specific environment. For the preview, add-ons are subscribed manually by {{< company-c8y >}}. If you want to add a new add-on to an environment, reach out to your Customer Success Manager or Account Executive, or contact [product support](/additional-resources/contacting-support/). A short description of each add-on is available.
```

> Each environment will be given a randomly generated name upon onboarding or creation. You can give a name and description to each environment by navigating to the respective tab at the top of the page. 

Reviewer: ```suggestion
Each environment gets a randomly generated name upon onboarding or creation. You can provide a name and description for each environment by navigating to the respective tab at the top of the page. 
```
